core/src/test/java/org/apache/accumulo/core/client/ZooKeeperInstanceTest.java

```diff
@@ -37,6 +37,7 @@
   private static final String IID_STRING = IID.toString();
   private ZooCacheFactory zcf;
   private ZooCache zc;
+  @SuppressWarnings("deprecation")
```

Comment:
Would be easier to deprecate this entire class than to suppress these warnings. This class will be removed when the ZKI it tests goes away.

Reply:
Fixed